extensions/tajo/README.md

```diff
@@ -0,0 +1,143 @@
+Deploying Apache Tajo™ on Google Cloud Platform
+===============================================
+
+Apache Tajo
+-----------
+
+Apache Tajo is a robust big data warehouse system. Dubbed “an SQL-on-Hadoop”, Tajo is optimized for running low-latency, scalable ad-hoc queries and ETL jobs on large-data sets stored on both HDFS and other data sources including Amazon S3 and Google Cloud Storage. By supporting SQL standards and leveraging advanced query optimization techniques, Tajo support both interactive analysis and complex ETL in a single solution. 
+
+This documents explains how to setup Tajo cluster on Google Cloud Platform using bdutil.
+
+Getting Started
+---------------
+
+1. Install gcloud SDK
+
+    https://cloud.google.com/sdk/
+    
+2. Install bdutil Tajo extension
+
+    $ git clone https://github.com/GoogleCloudPlatform/bdutil.git
+    
+3. Configure
+   
+    $ vi  bdutil_env.sh
+
+    ```
+    ############### REQUIRED ENVIRONMENT VARIABLES (no defaults) ##################
+
+    # A GCS bucket used for sharing generated SSH keys and GHFS configuration.
+    CONFIGBUCKET="YOUR_BUCKET" 
+
+    # The Google Cloud Platform text-based project-id which owns the GCE resources.
+    PROJECT="YOUR_PROJECT_ID" 
+
+    ###############################################################################
+
+    GCE_ZONE="us-central1-f"
+    
+    # change it to your instance type.
+    GCE_MACHINE_TYPE='n1-standard-4'  
+
+    # number of worker nodes 
+    NUM_WORKERS=2
+
+    # Prefix to be shared by all VM instance names in the cluster, as well as for
+    # SSH configuration between the JobTracker node and the TaskTracker nodes.
+    PREFIX='tajo'
+    ```
+    
+    $ vi extensions/tajo/tajo_env.sh
+    
+    ```
+    # path to tajo tarball 
+    TAJO_TARBALL_URI='gs://PATH_TO_TAJO_TARBALL/tajo-x.xx.x.tar.gz'
+    ```
+    
+4. Using cloudSQL for Tajo meta store (optional)
+    
+By default, Tajo stores its meta data in built-in Derby database in Tajo master node. Since it is ephemeral storage, you'd better use it for test purpose only. For continuous analysis work, using permanent meta store such as cloudSQL is strongly recommended.
+
+To use existing cloudSQL instance for Tajo meta store, set the instance id and connection information. 
+In case you set non-existing instance id, new cloudSQL instance will be created automatically. In this case, the instance id should be unique and userid should be 'root'.   
```

Comment:
Here and in the _env file, a WARNING that deleting the cluster will not remove any created cloud sql instances would be appreciated (as you would want for a persistent store).

Thinking about it a bit, would it make sense to include the instructions on how to create a new CloudSQL instance in the readme and the comments, but not actually create new cloud sql instances if they don't exist? I'm really paranoid about billing folks for things that they may have simply forgotten about. If the user actually invokes gcloud locally they will (hopefully) have a better understanding of the implications of using CloudSQL.